db/models/assessment_record/assessment_records.py

```diff
@@ -86,9 +86,15 @@ class AssessmentRecord(BaseModel):
     # These are defined as column_properties not as hybrid_property due to performance
     # Using column_property below forces the json parsing to be done on the DB side which is quicker than in python
     organisation_name = column_property(
-        func.jsonb_path_query_first(
-            jsonb_blob,
-            '$.forms[*].questions[*].fields[*] ? (@.key == "opFJRm").answer',
+        func.coalesce(
```

Comment:
As we have discussed, this approach is less than ideal going forward. I think the next best thing is adding it to the assessment_mapping_fund_round so at least we are maintaining the pointers to json fields in one place. One to add to tech debt i think - what do you think?

Reply:
yeah I understand, i think we just make a note that this is another area that needs refactoring 👍